pkgs/tools/misc/loadlibrary/default.nix

```diff
@@ -0,0 +1,30 @@
+{ fetchurl, cabextract, glibc_multi, fetchFromGitHub, readline, stdenv_32bit }:
+
+stdenv_32bit.mkDerivation {
```

Comment:
Because otherwise, the compilation fails with:

```
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/63gvnrj4z154kpyjpskl6s0hwmyx9x0w-glibc-2.25/lib/libm.so when searching for -lm
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/63gvnrj4z154kpyjpskl6s0hwmyx9x0w-glibc-2.25/lib/libm.so when searching for -lm
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: cannot find -lm
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/sqkgzdis17gvqlnxq5nkd9gm6hmn8rxz-gcc-5.4.0/lib/gcc/x86_64-unknown-linux-gnu/5.4.0/libgcc.a when searching for -lgcc
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: cannot find -lgcc
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/63gvnrj4z154kpyjpskl6s0hwmyx9x0w-glibc-2.25/lib/libgcc_s.so when searching for -lgcc_s
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/ncvznvhzbf226s5lqpv2a0345ba5f5hy-gcc-5.4.0-lib/lib/libgcc_s.so when searching for -lgcc_s
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/ncvznvhzbf226s5lqpv2a0345ba5f5hy-gcc-5.4.0-lib/lib/libgcc_s.so when searching for -lgcc_s
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/63gvnrj4z154kpyjpskl6s0hwmyx9x0w-glibc-2.25/lib/libgcc_s.so when searching for -lgcc_s
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: cannot find -lgcc_s
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/63gvnrj4z154kpyjpskl6s0hwmyx9x0w-glibc-2.25/lib/libc.so when searching for -lc
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/63gvnrj4z154kpyjpskl6s0hwmyx9x0w-glibc-2.25/lib/libc.so when searching for -lc
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: cannot find -lc
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/sqkgzdis17gvqlnxq5nkd9gm6hmn8rxz-gcc-5.4.0/lib/gcc/x86_64-unknown-linux-gnu/5.4.0/libgcc.a when searching for -lgcc
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: cannot find -lgcc
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/63gvnrj4z154kpyjpskl6s0hwmyx9x0w-glibc-2.25/lib/libgcc_s.so when searching for -lgcc_s
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/ncvznvhzbf226s5lqpv2a0345ba5f5hy-gcc-5.4.0-lib/lib/libgcc_s.so when searching for -lgcc_s
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/ncvznvhzbf226s5lqpv2a0345ba5f5hy-gcc-5.4.0-lib/lib/libgcc_s.so when searching for -lgcc_s
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: skipping incompatible /nix/store/63gvnrj4z154kpyjpskl6s0hwmyx9x0w-glibc-2.25/lib/libgcc_s.so when searching for -lgcc_s
/nix/store/zszmb7pksv3pqzm2nlpfgszd6mgqikc0-binutils-2.28/bin/ld: cannot find -lgcc_s
collect2: error: ld returned 1 exit status
make: *** [Makefile:27: mpclient] Error 1
```